app/models/room.rb

```diff
@@ -1,3 +1,10 @@
 class Room < ApplicationRecord
   belongs_to :hotel
+  has_many :guest_rooms
+  has_many :guests, through: :guest_rooms
+
+  def total_guests
+    self.guests.count
```

Comment:
Great encapsulation! It's helpful with this type of query to throw `.distinct` in there before the count just in case, but this US was not explicit about that so we accept either case!